website/content/docs/api-clients/client-agent.mdx

```diff
@@ -652,6 +652,17 @@ Learn more: https://cfl.re/CF_DNS_PROXY_FAILURE
 You can still install both the Cloudflare WARP client and the Boundary Client Agent on the same machine.
 As long as you don't run both at the same time, they should work as expected.
 
+### Huntress EDR client
+
+Host isolation and release via Huntress does not work as expected in Windows when used alongside the Boundary Client Agent.
+Huntress EDR client in Windows utilizes the Windows Filtering Platform (WFP) to isolate devices by blocking all non-essential traffic. This behavior conflicts with the Boundary Client Agent during device isolation.
```

Comment:
```suggestion
The Huntress EDR client for Windows uses the Windows Filtering Platform (WFP) to isolate devices by blocking all non-essential traffic. This behavior conflicts with the Boundary Client Agent during device isolation.
```

Reply:
Similarly, I'd recommend changing "utilizes" to "uses" since it is simple and more common.